FAQ.md

```diff
@@ -0,0 +1,55 @@
+FREQUENTLY ASKED QUESTIONS (FAQ) FOR LOGCAT READER
+
+![logcat](https://user-images.githubusercontent.com/35721447/39463089-c9967504-4d0d-11e8-8515-8acd63a7d732.png)
```

Comment:
Needs a fixed size since this appears as a huge image, consider using img tags:
``` <img src="/app/playstore_images/screenshots/screenshot-1.png" width="300px" /> ```

Reply:
there seems to be a problem on the pull, after resizing i still have huge image